x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js

```diff
@@ -33,7 +33,7 @@ export const SimpleTemplate = props => {
       {!color || color.length === 0 ? (
         <Fragment>
           <EuiFlexItem grow={false}>
-            <label>Color&nbsp;</label>
+            <span>Color&nbsp;</span>
```

Comment:
The point of this is increased accessibility, doesn't making this markup less semantic make it less accessible? Screen readers no longer understand that this is a label for something, it's now just some text on the page. 

Why remove the label? Wouldn't adding an id to the link control be the right change?

```diff
diff --git a/x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js b/x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js
index 4307ada682..66fb9edcae 100644
--- a/x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js
+++ b/x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js
@@ -33,10 +33,10 @@ export const SimpleTemplate = props => {
       {!color || color.length === 0 ? (
         <Fragment>
           <EuiFlexItem grow={false}>
-            <span>Color&nbsp;</span>
+            <label htmlFor="series-style">Color&nbsp;</label>
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiLink onClick={() => handlePlain('color', '#000000')}>
+            <EuiLink id="series-style" onClick={() => handlePlain('color', '#000000')}>
               Auto <EuiIcon type="bolt" />
             </EuiLink>
           </EuiFlexItem>
```

Reply:
@w33ble 

That is actually not a correct usage of a [label's `for` / `htmlFor`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for). You can only use it to assign an association with what they call a [`labelable` element](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable) otherwise it has no effect.

In this particular instance it's more appropriate to use `labelled-by` on link that contains the id of the descriptive element (which should not be a `<label>` element). That's not necessarily the scope of this particular PR, but it could be added if @bhavyarm thinks it would help.

Reply:
@cchaos thanks for the info, I wasn't aware of the labelable limitation. Sounds like a span here is what we want after all.